beacon-chain/main.go

```diff
@@ -3,10 +3,13 @@ package main
 
 import (
 	"fmt"
+	"io/ioutil"
 	"os"
 	"runtime"
 	runtimeDebug "runtime/debug"
 
+	"gopkg.in/yaml.v2"
+
```

Comment:
consider moving this import down (it stands in its own group), and go fmt.

Reply:
thanks, fixed